src/libstore/s3-binary-cache-store.md

```diff
@@ -3,7 +3,7 @@ R"(
 **Store URL format**: `s3://`*bucket-name*
 
 This store allows reading and writing a binary cache stored in an AWS S3 (or S3-compatible service) bucket.
-This store shares many idioms with the [HTTP Binary Cache Store](#http-binary-cache-store).
+This store shares many idioms with the [HTTP Binary Cache Store](./http-binary-cache-store.md).
```

Comment:
Weird because `src/libstore/http-binary-cache-store.md` does exist.

Reply:
```
   > error: File not found: ./http-binary-cache-store.md
       >      ┌─ command-ref/new-cli/nix3-help-stores.md:1073:40
       >      │
       > 1073 │ This store shares many idioms with the [HTTP Binary Cache Store](./http-binary-cache-store.md).
       >      │                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File not found: ./http-binary-cache-store.md
       >
       > Error: One or more incorrect links
```

Reply:
This is not how it works. The location is not resolved relative to the source file but where it ends up before getting processed by mdBook.

```suggestion
This store shares many idioms with the [HTTP Binary Cache Store](@docroot@/store/types/http-binary-cache-store.md).
```

Reply:
oops i saw relative paths being used, but i guess not under `src/libstore`. thanks for catching that ❤️